assignments/arrays.js

```diff
@@ -75,30 +75,50 @@ let inventory = [
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+// console.log("Car 33 is a " + inventory[32].car_year + " " + inventory[32].car_make + " " + inventory[32].car_model);
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`);
```

Comment:
This solution works but it is not dynamic; if the positions of the items are changed, the solution will fail. A better approach would be to loop through the array and return the item whose id is 33